src/components/AvatarWithIndicator.js

```diff
@@ -22,12 +23,17 @@ const propTypes = {
 
     /** The employee list of all policies (coming from Onyx) */
     policiesMemberList: PropTypes.objectOf(policyMemberPropType),
+
+    /** The user's wallet (coming from Onyx) */
+    ...userWalletPropTypes,
 };
 
 const defaultProps = {
     size: 'default',
     tooltipText: '',
     policiesMemberList: {},
+    // eslint-disable-next-line react/default-props-match-prop-types
+    userWallet: {},
```

Comment:
I think we can remove this lint disable, what if we updated the userWalletPropTypes to be export PropTypes.shape
and then we can use check that userwallet: userWalletPropTypes

Reply:
Not sure if there was a reason behind having userWalletPropTypes as:

```js
export default {
    /** User's wallet information */
    userWallet: PropTypes.shape({
        ...
    }),
};
```

vs

```js
export default PropTypes.shape({
    ...
});
```

But I've updated to the latter so we can avoid disabling the lint error: `eslint-disable-next-line react/default-props-match-prop-types`. Updated all usages of userWalletPropTypes as well.

cc: @marcaaron

Reply:
Great solution! I'm not sure if there was a clear reason tbh. I probably was just in the mood to use a spread operator. This seems like a bester "best practice".